This patch fixes an issue that was found in openscap 1.1.1+ when
libxml2 is upgraded to ver 2.9.2. It fixes the errors, that were
reported as entities already defined, showing up in the report
and guide generation.
This patch has not been contributed upstream, but is planned to be
done by 2015-Jan-31.

--- openscap-1.2.0/xsl/xccdf-report-impl.xsl.~1~	2014-09-26 01:41:04.068164604 -0700
+++ openscap-1.2.0/xsl/xccdf-report-impl.xsl	2014-12-10 10:51:48.443071716 -0800
@@ -49,7 +49,7 @@
     <xsl:param name="testresult"/>
     <xsl:param name="benchmark"/>
 
-    <div id="characteristics"><a name="characteristics"></a>
+    <div id="characteristics">
         <h2>Evaluation Characteristics</h2>
         <div class="row">
             <div class="col-md-5 well well-lg">
@@ -172,7 +172,7 @@
 <xsl:template name="compliance-and-scoring">
     <xsl:param name="testresult"/>
 
-    <div id="compliance-and-scoring"><a name="compliance-and-scoring"></a>
+    <div id="compliance-and-scoring">
         <h2>Compliance and Scoring</h2>
 
         <xsl:variable name="total_rules_count" select="count($testresult/cdf:rule-result[cdf:result])"/>
@@ -376,7 +376,7 @@
     <xsl:param name="benchmark"/>
     <xsl:param name="profile"/>
 
-    <div id="rule-overview"><a name="rule-overview"></a>
+    <div id="rule-overview">
         <h2>Rule Overview</h2>
 
         <div class="form-group js-only">
@@ -589,7 +589,6 @@
             </xsl:for-each>
         </div>
         <div class="panel-heading">
-            <a name="rule-detail-{generate-id($ruleresult)}"></a>
             <h3 class="panel-title">
                 <xsl:call-template name="item-title">
                     <xsl:with-param name="item" select="$item"/>
@@ -695,7 +694,7 @@
     <div class="js-only">
         <button type="button" class="btn btn-info" onclick="return toggleResultDetails(this)">Show all result details</button>
     </div>
-    <div id="result-details"><a name="result-details"></a>
+    <div id="result-details">
         <h2>Result Details</h2>
 
         <xsl:call-template name="result-details-inner-node">
--- openscap-1.2.0/xsl/xccdf-guide-impl.xsl.~1~	2014-09-26 01:41:04.067164602 -0700
+++ openscap-1.2.0/xsl/xccdf-guide-impl.xsl	2014-12-10 10:49:08.479661674 -0800
@@ -47,7 +47,7 @@
     <xsl:param name="benchmark"/>
     <xsl:param name="profile"/>
 
-    <div id="introduction"><a name="introduction"></a>
+    <div id="introduction">
         <div class="row">
             <div class="col-md-8 well well-lg">
                 <h2>
@@ -415,7 +415,7 @@
     <xsl:param name="benchmark"/>
     <xsl:param name="profile"/>
 
-    <div id="guide-tree"><a name="guide-tree"></a>
+    <div id="guide-tree">
         <h2>Checklist</h2>
 
         <table class="treetable table table-bordered">
@@ -434,7 +434,7 @@
     <xsl:param name="benchmark"/>
     <xsl:param name="profile"/>
 
-    <div id="rear-matter"><a name="rear-matter"></a>
+    <div id="rear-matter">
         <div class="row">
             <div class="col-md-12 well well-lg">
                 <xsl:if test="$benchmark/cdf:rear-matter">
